src/sagemaker/workflow/properties.py

```diff
@@ -109,6 +111,39 @@ def __getitem__(self, item: Union[int, str]):
         return self._items.get(item)
 
 
+class PropertiesMap(Properties):
+    """PropertiesMap for use in workflow expressions."""
+
+    def __init__(self, path: str, shape_name: str = None):
+        """Create a PropertiesMap instance representing the given shape.
+
+        Args:
+            path (str): The parent path of the PropertiesMap instance.
+            shape_name (str): The botocore sagemaker service model shape name.
+            root_shape_name (str): The botocore sagemaker service model shape name.
```

Comment:
`root_shape_name` is not present in the arguments. We should remove this from doc

Reply:
Removed